packages/app/README.md

```diff
@@ -191,18 +184,19 @@ The base URL of the H5Grove server.
 
 The path and/or name of the file to display in the UI.
 
-#### `axiosConfig?: AxiosRequestConfig` (optional)
-
-By default, `H5GroveProvider` does not make any assumption as to how to
-configure its internal [axios](https://axios-http.com/docs/req_config) client.
-If you use one of H5Grove's default API implementations, then you'll need to use
-this prop to pass the `file` query parameter as shown above.
+#### `fetcher?: Fetcher` (optional)
 
-If your API server requires authentication or is on a different domain, you'll
-need to pass the necessary request headers and configuration as well.
+An optional asynchronous function to fetch data and metadata from an h5grove
+back-end. The function accepts a URL together with some query parameters and
+options, and is expected to return a promise that resolves to an `ArrayBuffer`.
 
-> Remember to memoise or extract your `axiosConfig` object so the fetching cache
-> does not get cleared if/when your app re-renders.
```

Comment:
If I understand correctly, this warning still relevant for the `fetcher`. So I think it should appear here as well.